### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by michaelraczycki**
0.0.10 -> 0.0.11

### CODE REVIEW

File: pymc_experimental/version.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-0.0.10
+
```
Comment (twiecki) [inline pymc_experimental/version.txt:1]:
```suggestion
```

**Comment by ricardoV94** [comment]
I don't remember if we need to bump twice in case we already did a release and added the 0.11 tag like it happened now

**Comment by michaelraczycki** [comment]
I think I can just manually re-run the workflows after merging, and it should release to pypi

**Comment by michaelraczycki** [comment]
if that will turn out to be blocking for the release, I'd just delete the 0.0.11 (since it has only my bug fix) and release 0.0.11 again

**Comment by ricardoV94** [comment]
You can do that. You still have to delete the tag in the main branch: https://devconnected.com/how-to-delete-local-and-remote-tags-on-git/

**Pull request merged**